frontend/pages/hosts/details/cards/User/components/AddEndUserModal/AddEndUserModal.tsx

```diff
@@ -17,18 +17,26 @@ const AddEndUserModal = ({ onExit }: IAddEndUserModalProps) => {
     <Modal title="Add user" onExit={onExit} className={baseClass}>
       <>
         <div className={`${baseClass}__content`}>
-          <p>
-            Currently, <b>Username (IdP)</b> is only added when the host
-            automatically enrolls (ADE).{" "}
-          </p>
-          <p>
-            To add username when hosts enroll in the future, enable{" "}
-            <CustomLink
-              url={paths.CONTROLS_END_USER_AUTHENTICATION}
-              text="end user authentication"
-            />
-            .
-          </p>
+          <p>Currently, Username (IdP) can be added in the following ways:</p>
+          <ul style={{ listStyle: "disc", paddingLeft: "20px" }}>
+            <li style={{ marginBottom: "10px" }}>
```

Comment:
Note as reminder we'll want to move these in-line styles to the stylesheet in the next iteration